src/AWSCore.jl

```diff
@@ -203,7 +203,10 @@ Service endpoint URL for `request`.
 function service_url(aws, request)
     endpoint = get(request, :endpoint, request[:service])
     region = "." * aws[:region]
-    if endpoint == "iam" || (endpoint == "sdb" && region == ".us-east-1") || region=="."
+
+    regionless_service_endpoints = ["route53", "sdb"]
```

Comment:
Should be a tuple since it'll never change
```suggestion
    regionless_service_endpoints = ("route53", "sdb")
```